condabuild/meta.yaml

```diff
@@ -55,4 +56,4 @@ about:
     libraries.
   dev_url: https://github.com/BCG-X-Official/artkit
   doc_url: https://bcg-x-official.github.io/artkit/
-  doc_source_url: https://github.com/BCG-X-Official/artkit/blob/develop/README.rst
\ No newline at end of file
+  doc_source_url: https://github.com/BCG-X-Official/artkit/blob/develop/README.rst
```

Comment:
This URL doesn't even exist lol. Must've been missed since it's optional. Let's clean it up and set: https://github.com/BCG-X-Official/artkit/tree/1.0.x/sphinx/source